build.sbt

```diff
@@ -24,6 +24,16 @@ lazy val compilerSettings = CompilerSettings.options ++ Seq(
 
 lazy val commonSettings = projectSettings ++ coverageSettings ++ compilerSettings
 
+lazy val casper = (project in file("casper"))
+  .settings(commonSettings: _*)
+  .settings(
+    libraryDependencies ++= commonDependencies ++ protobufDependencies,
+    PB.targets in Compile := Seq(
+      scalapb.gen() -> (sourceManaged in Compile).value
```

Comment:
I don't know much about SBT, but comparing it to `comm`, there's a line 
```PB.gens.java -> (sourceManaged in Compile).value```
that looks like it has something to do with protobufs.  Do you need it here?

Reply:
I honestly don't know what `PB.gens.java` is for, but the `models` subproject doesn't use it and `casper` seems to compile fine without it. I tend to prefer minimal build settings and only adding extra things when necessary because something does not work.

Reply:
should be removed in comm i think, its an old thing, i will take care of it